.github/workflows/publish-docfx.yml

```diff
@@ -0,0 +1,39 @@
+
+name: Deploy DocFx
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
```

Comment:
Can we run this for PRs too, but just not commit the results?

Reply:
Sure! Added.